tooling-docker/src/main/java/org/jboss/eap/qe/ts/common/docker/Docker.java

```diff
@@ -153,18 +156,36 @@ protected boolean isRunning() throws Exception {
         return false;
     }
 
-    protected void stop() throws Exception {
+    public void stop() throws Exception {
         System.out.println(Ansi.ansi().reset().a("Stopping container ").fgCyan().a(name).reset()
                 .a(" with ID ").fgYellow().a(uuid).reset());
 
         new ProcessBuilder()
                 .command("docker", "stop", uuid)
                 .start()
                 .waitFor(10, TimeUnit.SECONDS);
+        terminateThreadPools();
+        removeDockerContainter();
+    }
+
+    public void kill() throws Exception {
```

Comment:
Where is this used at the moment? Or did you mean this is reserved for future usage?

Reply:
I use it in my tests for MP FT. Tests using it will come later.